example/test/Spec.hs

```diff
@@ -0,0 +1,2 @@
+main :: IO ()
```

Comment:
@felixminom since this is just a dummy project, I think we could get rid of `test` and the `test` target inside the `package.yaml`